tests/fixtures/repositories.py

```diff
@@ -27,6 +27,8 @@ def create_test_repository_payload() -> Dict[str, Any]:
         'url': 'https://repo.almalinux.org/',
         'tests_dir': 'almalinux/',
         'tests_prefix': '8.',
+        'team_id': 1,
+        'owner_id': 1,
```

Comment:
Just spotted this before merging. `owner_id` is no longer required, can you please remove it?